sherpa/ui/utils.py

```diff
@@ -12633,6 +12626,10 @@ def plot_model_component(self, id, model=None, replot=False,
         expression for a data set, including any instrument response.
         Use `plot_source_component` to display without any response.
 
+        .. versionchanged :: 4.13.2
```

Comment:
no planned 4.13.2 release- better to remove 'versionchanged'

Reply:
Done